### ISSUE TITLE
no Auth Provider found for name "oidc"

### ISSUE THREAD

**Issue by HaveFun83** [FairwindsOps/Nova#9]
### Installation Process
 
Github release on local-machine

### Nova Version

```
nova version 0.0.1
```

### Expected Behavior

Normal output

### Actual Behavior

```
Nova find --helm-version=auto
E0807 09:44:51.330033   27135 kube.go:56] Error creating kubernetes client: no Auth Provider found for name "oidc"
```

### Steps to Reproduce

enable the oidc provider on the kube-apiserver and try to run nova

**Comment by rbren** [comment]
Thanks for the report! Fixed in `1.0.3`

https://github.com/FairwindsOps/Nova/pull/15

**Comment by HaveFun83** [comment]
@rbren thanks a lot. Tested with nova version `2.0.1`
